tasks/configuration/Windows.yml

```diff
@@ -81,7 +81,7 @@
 #   when:
 #     - solr_auth_type is defined
```

Comment:
Should we remove it?
`solr_auth_type is defined` in defaults so it be always True